build.sbt

```diff
@@ -55,17 +58,19 @@ lazy val rootProject = (project in file("."))
     json4s,
     braveBackend,
     prometheusBackend,
-    tests
+    tests,
+    testServer
   )
 
 lazy val core: Project = (project in file("core"))
   .settings(commonSettings: _*)
   .settings(
     name := "core",
     libraryDependencies ++= Seq(
-      "org.scalacheck" %% "scalacheck" % "1.14.0" % "test",
+      "com.github.pathikrit" %% "better-files" % "3.4.0" % "test",
       scalaTest % "test"
-    )
+    ),
+    publishArtifact in Test := true // allow implementations outside of this repo
```

Comment:
if we are publishing the test artifact, it would be good to add some section in the documentation on developing new backends, that this artifact is available and can be used to test new backend impls

Reply:
Good idea